.github/workflows/r_ci.yml

```diff
@@ -30,13 +30,16 @@ jobs:
         uses: r-lib/actions/setup-r@v1
         with:
           r-version: ${{ matrix.r-version }}
-      - name: Install libcurl
-        run: sudo apt-get install libcurl4-openssl-dev
+      - name: Install linux dependencies  
+        run: |
+            sudo apt-get install libcurl4-openssl-dev
+            sudo apt-get install libudunits2-dev
+            sudo apt-get install libgdal-dev
       - name: Cache R packages
         uses: actions/cache@v2
         with:
           path: ${{ env.R_LIBS_USER }}
-          key: ${{ runner.os }}-r-2-
+          key: ${{ runner.os }}-r-3-
```

Comment:
@chinandrew does this need to be reverted?

Reply:
Don't think it matters (both caches are now created), but I reverting it makes sense since it's not needed right now.

Reply:
I changed it back and it still works.